app/assets/stylesheets/structure.scss

```diff
@@ -1,10 +1,55 @@
+*, *:before, *:after {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
```

Comment:
Looks like there's a [Compass mix-in](http://compass-style.org/examples/compass/css3/box_sizing/) for box-sizing, if you want to combine lines 2-4.